cpp/models/ode_secir/analyze_result.h

```diff
@@ -29,6 +29,9 @@
 namespace mio
 {
 
+namespace osecir
+{
+
 /**
  * @brief computes the p percentile of the parameters for each node.
```

Comment:
*Error*?

Not you fault, but when comparing with secirvvs I saw that

```
            param_percentil(
                node, [i](auto&& model) -> auto& {
                    return model.parameters.template get<TransmissionProbabilityOnContact>()[i];
                });
```

was missing under 'probs'. Or did I miss something now?

Could you do a quick comparison with the files in odesecirvvs generally (although there are clear difference for the more advanced model) but it would uncover possible things that we forgot to adapt.

Reply:
done